### PR TITLE
BL-71-fe-as-a-user-i-can-enter-a-search-query-to-search-asylum-case-decision-data

### PR DESCRIPTION
**Pull request by VWashingtonCoder**
## Contributors
* Vince Andre Washington (Author)
* Ryan Magro (Co-Author)

## Description
BL-71: FE-As A User I Can Enter A Search Query To Search Asylum Case Decision Data

* Built out a basic search bar that takes in a search term as well as a column category (via dropdown).
* Upon pushing the search/filter button, the table updates with any records matching the search term in the category passed.
* Table will continue to filter through any additional terms and categories added one at a time until the user obtains desired data or until the reset button is clicked. 

#### Video Link

[Loom Video] (https://www.loom.com/share/49c6a75f55c540019f93e5a2481ae050)

#### Jira Link
https://bloomtechlabs.atlassian.net/browse/BL-71?atlOrigin=eyJpIjoiMmE5NjFjMjgyNzdmNGFkYjliNzAzZTBlYTEzMTk2ZmQiLCJwIjoiaiJ9

## Type of change

- [X] New feature (non-breaking change which adds functionality)
- [X] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by vbrambila2 — DISMISSED** [review]
I like the layout of the search and the option to select categories. I think this makes more sense than having just the search bar. Also the reset button is great.

**Review by SafiaOmar — APPROVED** [review]
I like this way better its much UI also amazing readable code

**Review by XavierTwitty — APPROVED** [review]
This is nicely written I can easily read the actions and reducers. Good job using redux to manage state

File: src/components/pages/Table/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Form, Input, Button, Select } from 'antd';
+import { columns } from '../../../data/columns';
+import { connect } from 'react-redux';
+import { filterSearch, resetData } from '../../../state/actions';
```
Comment (Ricardorouge) [inline src/components/pages/Table/SearchBar.js:5]: don't think this line is needed since you're already grabbing this from props

**Review by Ricardorouge — APPROVED** [review]
looks good only problem I found was importing filtersearch resetData and grabbing them from props in the same file its not really a problem just redundant

**Review by EstrellaDionis — DISMISSED** [review]
Everything is easy to read and well though-out, great video as well!

**Review by EricMarshallLambert — APPROVED** [review]
I cannot speak to the JS, but this was nicely delivered. 

At this point will it just be a single search term? Did the filters take care of that?

**Pull request merged**